Classes/Aggregate/BackendPreviewAggregate.php

```diff
@@ -180,9 +208,17 @@ protected function getProcessedData(array \$databaseRow, array \$processedTcaCol
                 continue;
             }
             \$processedRow[\$field] = [];
+            \$i = 0;
             foreach (\$config['children'] as \$child) {
                 \$processedRow[\$field][] = \$this->getProcessedData(\$child['databaseRow'], \$child['processedTca']['columns']);
+                \$files = \$this->getFalFiles(\$processedRow[\$field][\$i]['uid'], 'tt_content', 'tx_' . \$processedRow[\$field][\$i]['CType'][0]);
+                if (count(\$files) > 0) {
+                    \$processedRow[\$field][\$i]['data_tx_' . \$processedRow[\$field][\$i]['CType'][0] . '_files'] = [];
+                    \$processedRow[\$field][\$i]['data_tx_' . \$processedRow[\$field][\$i]['CType'][0] . '_files'] = \$files;
```

Comment:
As all processed data is merged recursively, it is not a good idea to introduce a new namespace for the files